### ISSUE TITLE
Question about training loss L_p

### ISSUE THREAD

**Issue by xuhuazhe** [cvlab-epfl/single-stage-pose#4]
In the paper, the loss is \frac{1}{n} \sum ||(\hat{R}p_i + \hat{T} - (Rp_i+T))||. In the code, it is slightly different which is multiplied by intrinsic matrix before computing the distance. Are these two similar or mutually surrogate?

**Comment by YinlinHu** [comment]
@xuhuazhe In the paper, the error is defined as 3D reconstruction error, in the code, the error is defined as 2D reprojection error. Although 3D reconstruction error can lead to better ADD performance and 2D reprojection error can lead to better REP performance, the difference between them is pretty small in my experience.

**Comment by xuhuazhe** [comment]
Thank you! Good to know :)
